tracee-ebpf/tracee/tracee.go

```diff
@@ -458,7 +468,37 @@ func (t *Tracee) initTailCall(tailNum uint32, mapName string, progName string) e
 
 	return err
 }
+func getKallsymsData() ([]string, error) {
+	file, err := os.Open("/proc/kallsyms")
+
+	if err != nil {
+		return nil, fmt.Errorf("Could not open /proc/kallsyms")
+	}
+	scanner := bufio.NewScanner(file)
+	scanner.Split(bufio.ScanLines)
+	var text []string
 
+	for scanner.Scan() {
+		text = append(text, scanner.Text())
+	}
+	file.Close()
+	return text, nil
+}
+func getAddrFromKallsyms(func_name string) (uint64, error) {
+	text, _ := getKallsymsData()
```

Comment:
please handle errors - check if err != nil

Reply:
added the error handling